content/thirdpartynotices.txt

```diff
@@ -0,0 +1,32 @@
+This file is based on or incorporates material from the projects listed below
+(Third Party IP). The original copyright notice and the license under which
+Microsoft received such Third Party IP, are set forth below. Such licenses and
+notices are provided for informational purposes only. Microsoft licenses the
+Third Party IP to you under the licensing terms for the Microsoft product.
+Microsoft reserves all other rights not expressly granted under this agreement,
+whether by implication, estoppel or otherwise. 
+
+libuv v. 1.7.3
```

Comment:
Obviously we won't be able to send the PR and get the changes in in time for beta 8, so if we need the change, we'll need to hack it in.  Unless mainline scenarios are broken without the fix, we should probably go with the 1.7.4 as-is.   cc @DamianEdwards